src/api/app/jobs/published_repositories_cleanup_job.rb

```diff
@@ -0,0 +1,8 @@
+class PublishedRepositoriesCleanupJob < ApplicationJob
+  self.queue_adapter = :async if Rails.env.test?
```

Comment:
Why are you doing this? We have `config.active_job.queue_adapter = :inline` in the test environment.

Reply:
Explained in https://github.com/openSUSE/open-build-service/pull/10546#discussion_r556042023